opts/hosts_test.go

```diff
@@ -8,7 +8,7 @@ import (
 func TestParseHost(t *testing.T) {
 	invalid := map[string]string{
 		"anything":              "Invalid bind address format: anything",
-		"something with spaces": "Invalid bind address format: something with spaces",
+		"something with spaces": "parse tcp://something with spaces: invalid character \" \" in host name",
```

Comment:
looks like https://github.com/golang/go/commit/617c93ce740c3c3cc28cdd1a0d712be183d0b328 did it